### ISSUE TITLE
Make Query Tool Use fetch() to fill response

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#18314]
We need to stop refreshing the portlet each time we make a request, it should only populate the results (behave like the ES Search Portlet).

Migrate the current code to an endpoint.

If you hit the back button after clicking over a result it throws an error, should return without issues to the Query Tool Portlet.

**Comment by alfredo-dotcms** [comment]
IMO this is not a bug, this is default behavior of FF, this does not happen in Chrome.

Basically due to this configuration, https://github.com/dotCMS/core/blob/9a982f5991ee89ac0fe997f683b039f7b35578ab/dotCMS/src/main/webapp/html/common/top_inc.jsp#L20
the browser is not caching data for sensitive data in pages in the whole app, so FF throws a warning saying that the information can not be loaded (but offers a "Try Again" button so you can reload the content of that page).

![image](https://user-images.githubusercontent.com/37185433/90451649-82656780-e0a9-11ea-8a41-14776e2ac7a6.png)

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19327

**Comment by erickgonzalez** [comment]
**Note To Doc:**

New endpoint **/api/content/_search**, this is a POST and need a JSON with these possible values:
```
sort
limit
offset
userId -> userId to run the query to check the results a user will get (this is only for admins)
depth -> depth of the relationships
languageId
allCategoriesInfo
```

E.g
```
 curl --location --request POST 'http://localhost:8080/api/content/_search' \
      --header 'Content-Type: application/json' \
      --data-raw '{
           	 "query": "+contentType:webpagecontent",
            	 "sort":"modDate",
            	 "limit":20,
            	 "offset":0
     }'
```

**Comment by erickgonzalez** [comment]
PR for fix default values: https://github.com/dotCMS/core/pull/19448

**Comment by erickgonzalez** [comment]
Working as it should. Tested with several combinations of the possible params.

**Comment by bryanboza** [comment]
Fixed, tested the new endpoint and works as expected

**Comment by bryanboza** [comment]
We need to add more validations to the postman test, right now the 3 tests that we have is just validating if we return a 200 code, we need to validate something with the returned content or permissions in case of an anonymous request

**Comment by fmontes** [comment]
More test added but for master: https://github.com/dotCMS/core/pull/19461

**Comment by bryanboza** [comment]
Fixed, tested on release-20.10.1 // Postgres // FF. The changes in the tests are just in master not on this branch
